Zoo/Tour/TourOrTwoToursParallel.cs

```diff
@@ -0,0 +1,147 @@
+                }
+
+                if (places.Count == 1)
+                {
+                    return;
+                }
+
+                TourPlace tourPlace1 = places[0];
+                TourPlace tourPlacetourPlace2 = places[1];
+
+                if (tourPlace1.ZooZonesTypeZooZonesType == tourPlacetourPlace2.ZooZonesTypeZooZonesType)
+                {
+                    throw new TwoOfTheSameTourException($"There are two of {tourPlace1.ZooZonesTypeZooZonesType}");
+                }
+
+                bool isLegalCombination = false;
+
+
+
+                foreach (List<TourPlace> legalCombination in legallTourPlaces)
+                {
+                    BitArray bitArray = new(2);
+
+                    bitArray[0] = legalCombination[0].IsEqual(tourPlace1) && legalCombination[1].IsEqual(tourPlacetourPlace2);
+                    bitArray[1] = legalCombination[0].IsEqual(tourPlacetourPlace2) && legalCombination[1].IsEqual(tourPlace1);
+
+                    if (bitArray[0] || bitArray[1])
+                    {
+                        isLegalCombination = true;
+                        break;
+                    }
+                }
+
+                if (!isLegalCombination)
+                {
+                    throw new CannotMakeIllegalToursException($"Tour one {tourPlace1} cannot be parallel with {tourPlacetourPlace2}");
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+
+        public TourOrTwoToursParallel(List<TourPlace> places)
+        {
+            CheckIfTourCanBeParallel(places);
+            this.places = places;
+
+            touring = new Action(ActivateTour);
+        }
+
+        private void TourStart()
+        {
+            if (places.Count == 1)
+            {
+                Console.WriteLine($"Tour starts to {places[0]} ");
+                LogToFileSingleton.GetInstance().LogIntoEvent($"Tour starts to {places[0]} ");
+                return;
+            }
+
+            Console.WriteLine($"Tours start to {places[0]} and {places[1]}");
+            LogToFileSingleton.GetInstance().LogIntoEvent($"Tours start to {places[0]} and {places[1]}");
+
+        }
+
+        private void TourEnd()
+        {
+            if (places.Count == 1)
+            {
+                Console.WriteLine($"Tour ends to {places[0]} ");
+                LogToFileSingleton.GetInstance().LogIntoEvent($"Tour ends to {places[0]} ");
+                return;
+            }
+
+            Console.WriteLine($"Tours end to {places[0]} and {places[1]}");
+            LogToFileSingleton.GetInstance().LogIntoEvent($"Tours end to {places[0]} and {places[1]}");
+
+        }
+
+
+
+        private void ActivateTour()
+        {
+            TourStart();
+            int TenSecondsInMili = 10000;
+            // each tour is 10 seconds
+            Thread.Sleep(TenSecondsInMili);
```

Comment:
No need to call a variable by its size. If you change the interval you need to change the variable everywhere and this is a hassle. Call the variable by what it represents, like- `TourInterval` and put it in the consts file. There, you can clarify what it is with a comment(10 seconds).